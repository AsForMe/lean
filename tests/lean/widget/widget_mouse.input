{"seq_num": 0, "command": "sync", "file_name": "widget_mouse.lean"}
{"seq_num": 45, "command": "info", "file_name": "widget_mouse.lean", "line": 40, "column": 2}
{"seq_num": 46, "command": "get_widget", "file_name": "widget_mouse.lean", "line": 40, "column": 0, "id": 192}
{"seq_num": 47, "command" : "widget_event", "handler" : {"r" : [194,193,192]}, "kind": "onMouse", "file_name": "widget_mouse.lean", "line": 40, "column": 0, "id": 192}
