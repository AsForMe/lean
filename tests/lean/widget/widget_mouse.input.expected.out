{"msgs":[{"caption":"eval result","end_pos_col":0,"end_pos_line":13,"file_name":"widget_mouse.lean","pos_col":0,"pos_line":11,"severity":"information","text":"\"ba b--green\""}],"response":"all_messages"}
{"msgs":[{"caption":"eval result","end_pos_col":0,"end_pos_line":13,"file_name":"widget_mouse.lean","pos_col":0,"pos_line":11,"severity":"information","text":"\"ba b--green\""},{"caption":"trace output","file_name":"widget_mouse.lean","pos_col":0,"pos_line":40,"severity":"information","text":"successfully rendered widget\n"}],"response":"all_messages"}
{"message":"file invalidated","response":"ok","seq_num":0}
{"record":{"widget":{"column":0,"id":192,"line":40}},"response":"ok","seq_num":45}
{"response":"ok","seq_num":46,"widget":{"column":0,"html":{"c":[{"c":[{"a":null,"c":[{"c":[{"a":{"className":"ma3 pa3 ba b--red"},"c":["0","hello"],"e":{"onMouseEnter":{"h":0,"r":[194,193,192]}},"t":"div"}],"mouse_capture":true,"r":[194,193,192]},{"c":[{"a":{"className":"ma3 pa3 ba b--red"},"c":["0",{"c":[{"a":{"className":"ma3 pa3 ba b--red"},"c":["0",{"c":[{"a":{"className":"ma3 pa3 ba b--red"},"c":["0","asdf"],"e":{"onMouseEnter":{"h":0,"r":[198,196,195,193,192]}},"t":"div"}],"mouse_capture":true,"r":[198,196,195,193,192]},{"c":[{"a":{"className":"ma3 pa3 ba b--red"},"c":["0","qwerty"],"e":{"onMouseEnter":{"h":0,"r":[199,196,195,193,192]}},"t":"div"}],"mouse_capture":true,"r":[199,196,195,193,192]}],"e":{"onMouseEnter":{"h":0,"r":[196,195,193,192]}},"t":"div"}],"mouse_capture":true,"r":[196,195,193,192]},{"c":[{"a":{"className":"ma3 pa3 b--silver ba"},"c":[{"c":[{"a":{"className":"ma3 pa3 ba b--red"},"c":["0","foo",{"c":[{"a":{"className":"ma3 pa3 b--silver ba"},"c":["[world]"],"t":"div"}]}],"e":{"onMouseEnter":{"h":0,"r":[200,197,195,193,192]}},"t":"div"}],"mouse_capture":true,"r":[200,197,195,193,192]}],"t":"div"}]}],"e":{"onMouseEnter":{"h":0,"r":[195,193,192]}},"t":"div"}],"mouse_capture":true,"r":[195,193,192]},"0"],"t":"div"}]}]},"id":192,"line":40}}
{"record":{"status":"success","widget":{"column":0,"html":{"c":[{"c":[{"a":null,"c":[{"c":[{"a":{"className":"ma3 pa3 ba b--green"},"c":["1","hello"],"e":{"onMouseEnter":{"h":0,"r":[194,193,192]}},"t":"div"}],"mouse_capture":true,"r":[194,193,192]},{"c":[{"a":{"className":"ma3 pa3 ba b--red"},"c":["0",{"c":[{"a":{"className":"ma3 pa3 ba b--red"},"c":["0",{"c":[{"a":{"className":"ma3 pa3 ba b--red"},"c":["0","asdf"],"e":{"onMouseEnter":{"h":0,"r":[198,196,195,193,192]}},"t":"div"}],"mouse_capture":true,"r":[198,196,195,193,192]},{"c":[{"a":{"className":"ma3 pa3 ba b--red"},"c":["0","qwerty"],"e":{"onMouseEnter":{"h":0,"r":[199,196,195,193,192]}},"t":"div"}],"mouse_capture":true,"r":[199,196,195,193,192]}],"e":{"onMouseEnter":{"h":0,"r":[196,195,193,192]}},"t":"div"}],"mouse_capture":true,"r":[196,195,193,192]},{"c":[{"a":{"className":"ma3 pa3 b--silver ba"},"c":[{"c":[{"a":{"className":"ma3 pa3 ba b--red"},"c":["0","foo",{"c":[{"a":{"className":"ma3 pa3 b--silver ba"},"c":["[world]"],"t":"div"}]}],"e":{"onMouseEnter":{"h":0,"r":[200,197,195,193,192]}},"t":"div"}],"mouse_capture":true,"r":[200,197,195,193,192]}],"t":"div"}]}],"e":{"onMouseEnter":{"h":0,"r":[195,193,192]}},"t":"div"}],"mouse_capture":true,"r":[195,193,192]},"0"],"t":"div"}]}]},"id":192,"line":40}},"response":"ok","seq_num":47}
